src/ClaimService.Data.Provider.MsSql.Ef/ClaimServiceDbContext.cs

```diff
@@ -1,10 +1,21 @@
 ﻿using System.Threading.Tasks;
+using LT.DigitalOffice.ClaimService.Models.Db;
 using Microsoft.EntityFrameworkCore;
 
 namespace ClaimService.Data.Provider.MsSql.Ef;
 
 public class ClaimServiceDbContext : DbContext, IDataProvider
 {
+  public DbSet<DbClaim> Claims { get; set; }
+  public DbSet<DbChangeArgument> ChangeArguments { get; set; }
+  public DbSet<DbChangeTemplate> ChangeTemplates { get; set; }
+  public DbSet<DbClaimCategory> ClaimCategories { get; set; }
+  public DbSet<DbClaimCategoryExecutor> ClaimCategoryExecutors { get; set; }
```

Comment:
Maybe remove the "Claim" from the name "ClaimCategories" and "ClaimCategoryExecutors"?
Will remain "Categories" and "CategoryExecutors"